examples/workflow/4_analyze_results.ipynb

```diff
@@ -0,0 +1,415 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib.colors import LogNorm  # logarithmic z-axis\n",
+    "\n",
+    "fig, axs = plt.subplots(1, 2, figsize=(9, 4))\n",
+    "\n",
+    "plt.sca(axs[0])\n",
+    "axs[0].set_title('data')\n",
+    "dalitz_plot(frame_data, 'mSq_(3,4)', 'mSq_(2,4)', norm=LogNorm())\n",
+    "\n",
+    "plt.sca(axs[1])\n",
+    "axs[1].set_title('fit')\n",
+    "dalitz_plot(frame_phsp, 'mSq_(3,4)', 'mSq_(2,4)', norm=LogNorm())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Calculate fit fractions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The original intensity model that we optimized consists of several components. All registered component names can be retrieved via ``get_all_component_names()``:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "intensity_builder.get_all_component_names()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, these components are amplitudes and intensities that are added coherently or incoherently.\n",
+    "\n",
+    "In PWA, you are often interested in the fractions that those amplitudes contribute to the total intensity. These **fit fractions** can be calculated using the ``fit_fractions_with_propagated_errors()`` function. It requires amplitude or ``Intensity`` components that can be extracted from the ``IntensityBuilderXML`` instance. A nested list of the component names has to be specified as well. These are the names specified in the component XML attributes. If the inner lists contain more than one component, these components will be added coherently. In this way, you can calculate your own customized fit fractions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "components = intensity_builder.create_intensity_components([\n",
+    "    ['coherent_J/psi_-1_to_gamma_-1+pi0_0+pi0_0'],\n",
+    "    ['J/psi_-1_to_f2(1270)_-1+gamma_-1;f2(1270)_-1_to_pi0_0+pi0_0;'],\n",
+    "    ['J/psi_-1_to_f2(1270)_-2+gamma_-1;f2(1270)_-2_to_pi0_0+pi0_0;'],\n",
+    "    ['J/psi_-1_to_f2(1270)_0+gamma_-1;f2(1270)_0_to_pi0_0+pi0_0;'],\n",
+    "    [\n",
+    "        'J/psi_-1_to_f2(1270)_-1+gamma_-1;f2(1270)_-1_to_pi0_0+pi0_0;',\n",
+    "        'J/psi_-1_to_f2(1270)_-2+gamma_-1;f2(1270)_-2_to_pi0_0+pi0_0;',\n",
+    "        'J/psi_-1_to_f2(1270)_0+gamma_-1;f2(1270)_0_to_pi0_0+pi0_0;',\n",
+    "    ],\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last step is to call the actual fit fraction calculation function ``fit_fractions_with_propagated_errors()``. Here, the first required argument is a list of component pairs. Each pair resembles the nominator and denominator of a fit fraction."
```

Comment:
This function is quite complex, so here I would def link to the API

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/51b48a8618afb2997e753951df616771232a213c